### ISSUE TITLE
Can I get parameter of MT-Net/weights+final.h5 in current repo ?

### ISSUE THREAD

**Issue by jinmox99** [MahdiShAhmadi/ReDMark#19]
Your research is very interesting.

The paper has several parameters for generating MT-Net,

Can I get parameter for MT-Net/weights+final.h5 in current repo ?

**Comment by MahdiShAhmadi** [comment]
Hi, thank you for your kind words. If you mean hyper-parameters, Here is the set of hyper-parameters we used:
salt & pepper(4%) + Gaussian noise(std=3) + Jpeg(70) + smoothing(3x3))
loss 1:1
Dataset choice: cifar10 + resampled_pascal

The resulting h5 file in the repository is produced when the code was still in Tensorflow1. We tested it and it worked fine on the current setting (Tensorflow2)
